core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -115,7 +115,7 @@
 		'randomImage'                 => '{title_legend},name,headline,type;{source_legend},multiSRC,imgSize,fullsize,useCaption;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
 		'html'                        => '{title_legend},name,type;{html_legend},html;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests',
 		'rssReader'                   => '{title_legend},name,headline,type;{config_legend},rss_feed,numberOfItems,perPage,skipFirst,rss_cache;{template_legend:hide},rss_template;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
-		'twoFactor'                   => '{title_legend},name,headline,type;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
+		'two_factor'                   => '{title_legend},name,headline,type;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
```

Comment:
This change is wrong. Please revert.

Reply:
the name must match the controller, and the TwoFactorController automatically resolves to type two_factor. We could also override the controller type, but that seems "against" the best practice?

Reply:
What exactly do you mean by "automatically resolves to"? We are talking about `tl_module.type` here, which is not part of any Symfony naming convention.

Reply:
https://github.com/contao/contao/blob/master/core-bundle/src/DependencyInjection/Compiler/RegisterFragmentsPass.php#L133-L147

Reply:
Ah, nice. 👍